types/styled-system/index.d.ts

```diff
@@ -1,14 +1,60 @@
-// Type definitions for styled-system 2.3
+// Type definitions for styled-system 3.0
 // Project: https://github.com/jxnblk/styled-system#readme
 // Definitions by: Marshall Bowers <https://github.com/maxdeviant>
 //                 Ben McCormick <https://github.com/phobon>
 //                 Justin Bennett <https://github.com/zephraph>
 //                 Christopher Pappas <https://github.com/damassi>
 //                 Eloy Durán <https://github.com/alloy>
 //                 Matthieu Vachon <https://github.com/maoueh>
+//                 Adam Lavin <https://github.com/lavoaster>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.6
 
+export namespace util {
```

Comment:
Using namespaces is discouraged. If this is another module in the package then you should add another ambient declaration file for it that follows the same FS layout as the package uses.

Reply:
Like it was before this commit? https://github.com/DefinitelyTyped/DefinitelyTyped/pull/27683/commits/a4f5821903a4f1d8f9dd9823a00361ca2bb297cd

The reasoning for that change is that `styled-system/utils` isn't a valid import, which why I brought it back into one file.

Reply:
Is it `styled-system/util` or something like `styled-system/dist/util`?

Reply:
The latter.

Reply:
Ok, then that’s what the typings should follow 👍

Reply:
Alright